src/Security/Authentication/test/CertificateTests.cs

```diff
@@ -855,17 +854,29 @@ public async Task VerifyValidationResultNeverCachedAfter30Min(bool cache)
                             options.RevocationMode = configureOptions.RevocationMode;
                             options.ValidateValidityPeriod = configureOptions.ValidateValidityPeriod;
                             options.AdditionalChainCertificates = configureOptions.AdditionalChainCertificates;
+                            options.TimeProvider = configureOptions.TimeProvider;
+
+                            if (timeProvider != null)
+                            {
+                                options.TimeProvider = timeProvider;
+                            }
```

Comment:
```suggestion
                            options.TimeProvider = timeProvider  ?? configureOptions.TimeProvider;
```

Seems easier to comprehend, you can ignore if you want.

Reply:
Agree

Reply:
Makes sense, but I'm not going to burn another CI cycle for it.